pyproject.toml

```diff
@@ -27,6 +27,7 @@ dependencies = [
     "spurplus ~= 2.3.5",
     "websockets ~= 10.3",
     "charset_normalizer ~= 2.1.1",
+    "pandas",
```

Comment:
Is it used by csv parser only? Try built-in csv lib. We need to minimize dependencies on LISA core parts.

https://docs.python.org/3/library/csv.html